pkg/maps/ctmap/types.go

```diff
@@ -487,12 +487,14 @@ type CtEntry struct {
 func (c *CtEntry) GetValuePtr() unsafe.Pointer { return unsafe.Pointer(c) }
 
 const (
-	RxClosing  = 1 << 0
-	TxClosing  = 1 << 1
-	Nat64      = 1 << 2
-	LBLoopback = 1 << 3
-	SeenNonSyn = 1 << 4
-	NodePort   = 1 << 5
+	RxClosing     = 1 << 0
+	TxClosing     = 1 << 1
+	Nat64         = 1 << 2
+	LBLoopback    = 1 << 3
+	SeenNonSyn    = 1 << 4
+	NodePort      = 1 << 5
+	ProxyRedirect = 1 << 6
+	DSR           = 1 << 7
```

Comment:
Nit: could use `iota` to define this and make future additions easier:

```suggestion
	RxClosing     = 1 << iota
	TxClosing
	Nat64
	LBLoopback
	SeenNonSyn
	NodePort
	ProxyRedirect
	DSR
```

Reply:
Ah yeah, good point, will change it into this style.